src/native/libs/System.Native/pal_time.c

```diff
@@ -95,22 +95,22 @@ int32_t SystemNative_GetCpuUtilization(ProcessCpuInformation* previousCpuInfo)
     uint64_t lastRecordedKernelTime = previousCpuInfo->lastRecordedKernelTime;
     uint64_t lastRecordedUserTime = previousCpuInfo->lastRecordedUserTime;
 
-    uint64_t cpuTotalTime = 0;
+    double cpuTotalTime = 0.0;
```

Comment:
```suggestion
    uint64_t cpuTotalTime = 0;
```
In the calculation below `cpuBusyTime * 100.0 / cpuTotalTime` it is sufficient to have any of the values using double precision to get a double result. Changing these other intermediate values to double is triggering the errors about implicit conversions that lose precision. double can't represent large integers exactly because it loses precision the larger it gets. For values bigger than 2^53 only even integers can be represented exactly, above 2^54 only multiples of 4 can be represented, and so on. In this case (currentTime - lastRecordedCurrentTime) on line 101 is a uint64_t typed expression so assigning it to a variable of type double triggers the warning.